### ISSUE TITLE
Need a couple of examples in README.md

### ISSUE THREAD

**Issue by vors** [Jaykul/Reflection#2]
Would be great to see sneak peek of functionality right in the README.md.

**Comment by Jaykul** [comment]
Yep -- we're, uhm, working on some tools in PoshCode/ModuleBuilder to generate stuff like that, so I'm trying to avoid doing the work by hand ;-) (Right @jrich523?)

**Comment by vors** [comment]
That's ambitious :) automate all the things!
